pvsystemprofiler/scripts/longitude_script.py

```diff
@@ -60,7 +65,8 @@ def evaluate_systems(df, df_ground_data, power_column_label, site_id, time_shift
                     if manual_time_shift == 1:
                         dh.fix_dst()
```

Comment:
handling of the data handler needs to be fixed. currently, this data handler from the outer scope does not get the pipeline run on it